javascript-carousel/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Carousel</title>
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="slider">
+    <div class="img-box">
+      <img src="images/001.png" alt= 'green pokemon' class="slide">
+    </div>
+    <div class="prev">
+      <button class="btn" onclick="prev()">
```

Comment:
You should not be using inline event handlers like this. Use `addEventListener` in your JavaScript file